src/webui/www/private/scripts/client.js

```diff
@@ -735,6 +744,22 @@ window.addEvent('load', function() {
         registerMagnetHandler();
     });
 
+    $('showFiltersSidebarLink').addEvent('click', function(e) {
+        showFiltersSidebar = !showFiltersSidebar;
+        localStorage.setItem('show_filters_sidebar', showFiltersSidebar.toString());
```

Comment:
I find it a bit messy to have direct localStorage used directly in code, having a central place such as `Preference` class defining getters/setters would be neater IMO. Of course this is just a suggestion for a later PR/cleanup.

Reply:
Great suggestion, I completely agree.